src/api.js

```diff
@@ -136,21 +136,20 @@ exports.rmdir = function(path, callback) {
 
 
 exports.exists = function(path, callback) {
-  var success = wrapSuccess(callback)
-    , fail = wrapFail(callback);
+  var fail = wrapFail(callback);
```

Comment:
@haadcode managing to sort this merge now. I think this variable can actually be removed completely based on your fix?